addons/web/static/src/js/widgets/rainbow_man.js

```diff
@@ -47,9 +47,9 @@ var RainbowMan = Widget.extend({
         // this is done in a setTimeout to prevent the click that triggered the
         // rainbow man to close it directly
         setTimeout(function () {
-            core.bus.on('click', self, function (ev) {
+            self.$el.parent().on('mousedown', function (ev) {
```

Comment:
Remind me why that change ?

Reply:
in order to stop propagation and catch original event both events should be same. before this, last click will destroy rainbowman too
because click event is covered in mousedown but it is not same event so it will not stop propagation.
another thing is here we can use $('body') instead of self.$el.parent() i guess